app/views/shared/_client_primary_info_fields.html.erb

```diff
@@ -48,33 +48,26 @@
             t("hub.clients.show.date_of_birth"),
             help_text: t("hub.clients.show.date_of_birth_help"),
             ) %>
-      <%= f.cfa_select(:primary_tin_type, "Identification Type", tin_options_for_select(include_ssn_no_employment: true, include_itin: true)) %>
+    <% end %>
+    <%= f.cfa_select(:primary_tin_type, "Identification Type", tin_options_for_select(include_ssn_no_employment: true, include_itin: true)) %>
+    <%= f.cfa_input_field(
+        :primary_ssn,
+        t("general.ssn_itin"),
+        type: :tel,
+        classes: ["form-width--name"],
+        options: { maxlength: 11 }
+    ) %>
+    <% unless f.object.client&.persisted? %>
       <%= f.cfa_input_field(
-            :primary_ssn,
-            t("general.ssn_itin"),
+            :primary_ssn_confirmation,
+            "Re-enter SSN/ITIN",
             type: :tel,
             classes: ["form-width--name"],
             options: { maxlength: 11 }
-          ) %>
-      <% unless f.object.client&.persisted? %>
-        <%= f.cfa_input_field(
-              :primary_ssn_confirmation,
-              "Re-enter SSN/ITIN",
-              type: :tel,
-              classes: ["form-width--name"],
-              options: { maxlength: 11 }
-          ) %>
-      <% end %>
+      ) %>
+    <% end %>
+    <% if is_ctc %>
```

Comment:
It seems to me that this `if is_ctc` isn't related to this pull request, but is also definitely a good idea. Is that accurate? No change requested, just curious.

Reply:
You need to use the expander to see how the full diff was affected here. There is an unchanged line for "is_ctc" that starts on line 45. i've now added an end on line 51 so that only that part is is_ctc. Now, the full ssn section is shared between both types of intakes. But we do not need IP PIN for GYR, so that is wrapped in a is_ctc section.